.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,13 +2,44 @@ Fixes #
 
 ## Changes
 
-- x
-- y
-- z
+- _Insert list of changes_
 
-## Checks
+## 📸 Looks like
 
-Delete if not applicable:
+_Insert screenshots and/or videos here_
+
+## How to QA
+
+_Update these checklists as necessary_
+
+### 👀 Visual changes
+
+_Insert a detailed list explaining exactly how to QA your changes from a designer’s PoV_
+
+- `git checkout main`
+- `git pull`
+- `git checkout branch-name`
```

Comment:
I have to run for a meeting... Just one quick thought: if this PR comes from a fork, than the checkout isn't as easy as that. One would need to add the origin.

Maybe using the GitHub CLI is easier after all. It would be something like `gh pr checkout 756`. But it would require people to have the CLI installed and set up.

Reply:
Hmmm, I’ve never dealt with forks, is this a case we need to cover?